Makefile

```diff
@@ -178,6 +182,17 @@ endif
 	docker build --pull -t $(BUILD_IMAGE):latest-$(ARCH) . --build-arg BIRD_IMAGE=$(BIRD_IMAGE) --build-arg QEMU_IMAGE=$(CALICO_BUILD) --build-arg GIT_VERSION=$(GIT_VERSION) -f ./Dockerfile.$(ARCH)
 	touch $@
 
+# download BIRD source to include in image.
+$(BIRD_SOURCE):
```

Comment:
These two targets have no dependencies so they won't be rebuilt when bird/felix is updated.  Could you depend on `go.mod`?